09LessonNine/in-class-project/css/style.css

```diff
@@ -0,0 +1,99 @@
+main {
+    text-align: center;
+    width: 100vw;
+    height: 100vh;
+    background: url(../img/1.jpg);
+    background-size: cover;
+}
+.opacity {
```

Comment:
great use of the opacity! remember to utilize it on the services page as well though.